src/event/event_kevent.c

```diff
@@ -2363,6 +2363,12 @@ _dispatch_event_loop_timer_arm(dispatch_timer_heap_t dth, uint32_t tidx,
 		target += range.leeway;
 		range.leeway = 0;
 	}
+#if !NOTE_ABSOLUTE
```

Comment:
This won't be defined on all platforms (e.g. Windows).

Reply:
Right, but Windows is using src/event/event_windows.c. This entire file gets #ifdef'd out on Windows.